package.json

```diff
@@ -3,9 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bootstrap": "^4.1.3",
+    "eslint": "^5.6.1",
```

Comment:
for some reason it is also in `devDependencies` (and it should be there). Could you remove it from `dependencies`?.